docs/data_management/tutorial/globus_group_tutorial.md

```diff
@@ -0,0 +1,223 @@
+# Setting up Globus as a Research Manager for your Group
+
+This tutorial is designed to guide UAB research managers, such as Lab PIs, Core Directors, and their management staff, to help them set up and use Globus Connect Personal (GCP) for secure data sharing on their local/personal computers. GCP allows you to share data with collaborators efficiently meeting security and compliance standards for file sharing.
+
+**What You Will Learn:**
+
+1. [What is Globus, and why you should use it?](globus_group_tutorial.md#what-is-globus-and-why-you-should-use-it)
+1. [How do you get onto Globus?](globus_group_tutorial.md#how-do-you-get-onto-globus)
+1. [Installing Globus Connect Personal on Windows](globus_group_tutorial.md#installing-gcp-on-windows)
+1. [Installing Globus Connect Personal on MacOS](globus_group_tutorial.md#installing-gcp-on-macos)
+1. [Configuring the GCP endpoint to share specific folders](globus_group_tutorial.md#configuring-the-globus-connect-personal-endpoint-to-share-specific-folders)
+1. [Configuring and Sharing a Collection for your endpoint](globus_group_tutorial.md#configuring-and-sharing-a-collection-for-your-endpoint)
+
+## What is Globus, and why you should use it?
+
+We have a guide here(./globus_individual_tutorial.md#what-is-globus) that explains Globus, and its features.
+
+## How do you get onto Globus?
+
+We have a guide here(./globus_individual_tutorial.md#how-do-you-get-on-globus) that shows how to access Globus.
+
+## Installing Globus Connect Personal
+
+Globus Connect Personal is available on the following Operating Systems, Linux, MacOS and Windows. The steps in this tutorial are curated from the Globus docs showing how to install Globus on [Windows](https://docs.globus.org/globus-connect-personal/install/windows/) and [MacOS](https://docs.globus.org/globus-connect-personal/install/mac/).
+
+### Installing GCP On Windows
+
+The following steps will guide you to install `GCP` on your local machine running the `Windows OS`.
+
+1. Navigate to the [Globus Connect Personal page](https://www.globus.org/globus-connect-personal), scroll to look for the GCP Windows version. Click on the **`INSTALL NOW`** button highlighted red in the image below.
```

Comment:
Use "Double Quotes" for GUI element text instead of **bold** or `verbatim/monospace` markup. The text in GUIs isn't bold or monospaced usually. And in English the "Double Quotes" indicate text comes from another source. In this case, the other source is the GUI itself.

Reply:
This is done